tests/basic.ispec.js

```diff
@@ -4,176 +4,297 @@ import { createHash } from "crypto";
-  const path = [44, 474, 5, 0, 3];
-  const resp = await app.getAddressAndPubKey(path);
+    console.log(resp);
 
-  console.log(resp);
+    expect(resp.return_code).toEqual(0x9000);
+    expect(resp.error_message).toEqual("No errors");
 
-  expect(resp.return_code).toEqual(0x9000);
-  expect(resp.error_message).toEqual("No errors");
+    expect(resp).toHaveProperty("bech32_address");
+    expect(resp).toHaveProperty("pk");
 
-  expect(resp).toHaveProperty("bech32_address");
-  expect(resp).toHaveProperty("pk");
+    expect(resp.bech32_address).toEqual("oasis1qphdkldpttpsj2j3l9sde9h26cwpfwqwwuhvruyu");
+    expect(resp.pk.length).toEqual(32);
+  });
 
-  expect(resp.bech32_address).toEqual("oasis1ectg9qs5f7vlnlxce96gm264tqjng0glydhhnk2h4e4tgn442lgtwzrun");
-  expect(resp.pk.length).toEqual(32);
-});
+  test("showAddressAndPubKey_Ed25519", async () => {
+    const app = new OasisApp(transport);
 
-test("showAddressAndPubKey", async () => {
-  jest.setTimeout(60000);
+    // Derivation path. First 3 items are automatically hardened!
+    const path = [44, 474, 5, 0, 3];
+    const resp = await app.showAddressAndPubKey_ed25519(path);
 
-  const transport = await TransportNodeHid.create(1000);
-  const app = new OasisApp(transport);
+    console.log(resp);
 
-  // Derivation path. First 3 items are automatically hardened!
-  const path = [44, 474, 5, 0, 3];
-  const resp = await app.showAddressAndPubKey(path);
+    expect(resp.return_code).toEqual(0x9000);
+    expect(resp.error_message).toEqual("No errors");
 
-  console.log(resp);
+    expect(resp).toHaveProperty("bech32_address");
+    expect(resp).toHaveProperty("pk");
 
-  expect(resp.return_code).toEqual(0x9000);
-  expect(resp.error_message).toEqual("No errors");
+    expect(resp.bech32_address).toEqual("oasis1qphdkldpttpsj2j3l9sde9h26cwpfwqwwuhvruyu");
+    expect(resp.pk.length).toEqual(32);
+  });
 
-  expect(resp).toHaveProperty("bech32_address");
-  expect(resp).toHaveProperty("pk");
+  test("getAddressAndPubKey_Secp256k1", async () => {
+    const app = new OasisApp(transport);
 
-  expect(resp.bech32_address).toEqual("oasis1ectg9qs5f7vlnlxce96gm264tqjng0glydhhnk2h4e4tgn442lgtwzrun");
-  expect(resp.pk.length).toEqual(32);
-});
+    // Derivation path. First 3 items are automatically hardened!
+    const path = [44, 60, 0, 0, 0];
+    const resp = await app.getAddressAndPubKey_secp256k1(path);
 
-test("appInfo", async () => {
-  const transport = await TransportNodeHid.create(1000);
-  const app = new OasisApp(transport);
+    console.log(resp);
 
-  const resp = await app.appInfo();
+    expect(resp.return_code).toEqual(0x9000);
+    expect(resp.error_message).toEqual("No errors");
 
-  console.log(resp);
+    expect(resp).toHaveProperty("hex_address");
+    expect(resp).toHaveProperty("pk");
 
-  expect(resp.return_code).toEqual(0x9000);
-  expect(resp.error_message).toEqual("No errors");
+    expect(resp.hex_address).toEqual("7dac3ca6b185b7778a7027a52c0cc3940d2e1ca0");
```

Comment:
The standard BIP44 address for testing mnemonic `equip will roof matter pink blind book anxiety banner elbow sun young` is `95e5e3C1BDD92cd4A0c14c62480DB5867946281D`. Where does discrepancy come from?